src/vm/excep.cpp

```diff
@@ -7784,11 +7783,19 @@ LONG WINAPI CLRVectoredExceptionHandler(PEXCEPTION_POINTERS pExceptionInfo)
     //
     return retVal;
 #else // !FEATURE_PAL
+
+#if defined(WIN64EXCEPTIONS) && defined(FEATURE_HIJACK)
```

Comment:
I am sorry, I have confused myself into thinking the change is already visible in this diff.